RELEASES.md

```diff
@@ -79,4 +90,5 @@ deadline of 3 weeks.
 | 1.22.0  | 2022/04/15 | 2022/04/15 |    0 days  | 2023/04/15  |
 | 1.23.0  | 2022/07/15 |            |            |             |
 
+TODO(RyanTheOptimist): Should we also mention the security release schedule here? I think that would be helpful.
```

Comment:
Yeah. I don't think we have an explicit schedule, since of course there could be an emergency / 0-day that needs attention, but historically we do 1 per quarter, about midway into the major release cycle.

Reply:
Thanks. I've added a short blurb to this effect.